module2/lessons/forms.md

```diff
@@ -347,63 +399,67 @@ Can you guess what our next error is going to be?
 
 Run your tests, and you should see something like this:
 
+```bash
 Failure/Error: click_button 'Delete'
-
-```
 ActionController::RoutingError:
   No route matches [DELETE] "/artists/34"
 # ./spec/features/artists/delete_spec.rb:9:in `block (2 levels) in <top (required)>'
 ```
 
-No problem - we have seen errors like this before; our application doesn't know how to handle this request, so let's update our routes.rb:
+No problem - we have seen errors like this before; our application doesn't know how to handle this request, so let's update our routes.rb.
+
+**config/routes.rb**
 
 ```ruby
-# config/routes.rb
 Rails.application.routes.draw do
-
-  get '/songs', to: 'songs#index'
-  get '/songs/:id', to: 'songs#show'
-  get '/artists', to: 'artists#index'
-  get '/artists/new', to: 'artists#new'
-  post '/artists', to: 'artists#create'
-  delete '/artists/:id', to: 'artists#destroy'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "/songs", to: "songs#index"
+  get "/songs/:id", to: "songs#show"
+  get "/artists/:artist_id/songs", to: "artist_songs#index"
+  get "/artists", to: "artists#index"
+  get "/artists/new", to: "artists#new"
+  post "/artists", to: "artists#create"
+  delete "/artists/:id", to: "artists#destroy"
 end
 ```
 
-Now, we have a route which will respond to a `DELETE` with the path of `'/artists/2' or 'artists/45'` where 2 and 45 are ids of existing artists.
+Now, we have a route which will respond to a `DELETE` with the path of `'/artists/2' or 'artists/45'` where 2 and 45 are ids of existing artists.
 
-Finally, our test will tell us to go create that `destroy` action on our artists controller:
+Finally, our test will tell us to go create that `destroy` action on our artists controller.
 
-```ruby
-# app/controllers/artists_controller.rb
+**app/controllers/artists_controller.rb**
 
+```ruby
 class ArtistsController < ApplicationController
   def index
+    @artists = Artist.all
   end
-
+  
   def new
   end
 
   def create
-    artist = Artist.create(artist_params)
-    redirect_to '/artists'
+    Artist.create(name: params[:name])
```

Comment:
Since this is the third spot you've changed it, I think I must just be missing something as to why we aren't using strong params here anymore.